docs/resources/oxide_silo.md

```diff
@@ -0,0 +1,85 @@
+---
+page_title: "oxide_silo Resource - terraform-provider-oxide"
+---
+
+# oxide_silo (Resource)
+
+This resource manages the creation of an Oxide Silo.
+
+Note: On Silo updates the only attribute that can be updated is the `quotas`. The rest of the attributes will cause a destroy first and then another create.
```

Comment:
```suggestion
-> Only the `quotas` attribute supports in-place modification. Changes to other attributes will result in the silo being destroyed and created anew.
```

The `->` at the beginning will allow this to be rendered as a panel on the website.

![image](https://github.com/user-attachments/assets/51f82cf4-d58e-444f-88e8-6af7146e551c)